File: benchmark/results/java-multilang/run-001/b-baseline/output/com/metaengine/demo/billing/value_objects/PaymentTerms.java
package com.metaengine.demo.billing.value_objects;

import java.time.Instant;

/** PaymentTerms value object for the billing domain. */
public record PaymentTerms(
    String id,
    Instant createdAt,
    Instant updatedAt,
    String name,
    String description
) {}

File: benchmark/results/python-multilang/run-005/a-mcp/output/billing/value_objects/payment_terms.py
from datetime import datetime

"""PaymentTerms value object."""
class PaymentTerms:
    def __init__(self, id: str, created_at: datetime, updated_at: datetime, name: str, description: str):
        self.id = id
        self.created_at = created_at
        self.updated_at = updated_at
        self.name = name
        self.description = description

